com.avocode.Avocode.yml

```diff
@@ -0,0 +1,54 @@
+id: com.avocode.Avocode
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+base: org.electronjs.Electron2.BaseApp
+base-version: "19.08"
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --system-talk-name=org.freedesktop.login1
+  - --system-talk-name=org.bluez
+  - --filesystem=home
+command: avocode
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /share/man
+modules:
+
+  - name: avocode
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 avocode.sh /app/bin/avocode
+      - install -Dm755 apply_extra.sh /app/bin/apply_extra
+      - install -Dm755 /usr/bin/desktop-file-edit -t /app/bin/
+      - install -Dm755 /usr/bin/ar -t /app/bin/
+      - install -Dm755 /usr/lib/$(gcc -print-multiarch)/libbfd-*.so -t /app/lib/
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
+    sources:
```

Comment:
This isn't a formal requirement *yet*, but can you add something along what Zoom does here: https://github.com/flathub/us.zoom.Zoom/blob/master/us.zoom.Zoom.json#L72? If upstream provides Debian/Ubuntu repo, external-data-checker can consume that as well: https://github.com/endlessm/flatpak-external-data-checker/#debian-repo-checker

Reply:
done